<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Images/images.qrc">
    <normaloff>:/pics/applogo.png</normaloff>:/pics/applogo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#LoginForm{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="loginbox" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#loginbox{
background-color: qlineargradient(spread:pad, x1:0.181818, y1:0.017, x2:0.818, y2:0.971591, stop:0.0568182 rgba(0, 120, 207, 255), stop:1 rgba(6, 40, 61, 255));
}

QWidget#loginbox{
border-radius: 50%;
}</string>
   </property>
   <widget class="QLabel" name="username">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="uname">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border:2px solid black;
background-color:rgb(239, 245, 245);
font: 12pt &quot;Times New Roman&quot;;
border-radius:10px;
background-color: rgb(192, 220, 243);
}

QLineEdit:focus{
border:2px solid rgb(255, 255, 255);

}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwd">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border:2px solid black;
background-color:white;
font: 12pt &quot;Times New Roman&quot;;
border-radius:10px;
background-color: rgb(192, 220, 243);
}

QLineEdit:focus{
border:2px solid rgb(255, 255, 255);

}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="password">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="showmsg">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: rgb(214, 228, 229);
font: 75 12pt &quot;Arial&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/user.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="LoginBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:0px rgb(255, 255, 255);
background-color:#06283D;
color:rgb(255, 255, 255);
font: 75 12pt &quot;Times New Roman&quot;;
border-radius:10px;

}

QPushButton:focus:{
border:1px solid rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
